bond/bond.go

```diff
@@ -77,28 +77,28 @@ func getLinkObjectsFromConfig(bondConf *bondingConfig, netNsHandle *netlink.Hand
 		}
 		linkNames = append(linkNames, s)
 	}
+
+	// currently supporting two or more links to one bond.
+	if len(linkNames) < 2 {
+		return nil, fmt.Errorf("Bonding requires at least two links, we have %+v", len(linkNames))
```

Comment:
Please, either use `%+v` for `linkNames` or `%d` with `len(linkNames)`. I'd go for the former, as it will be easier to debug in case of error

Reply:
This is the convention used on this project. Regardless of the type, %+v is always used. I'm planning to create a pr to change this a bit.